keycloak/keycloak-admin-ui
# Realm setting profiles

## Motivation
1. Add radio button switch between "From view" and "JSON editor" as per https://marvelapp.com/prototype/h11728e/screen/81529504 and https://marvelapp.com/prototype/h11728e/screen/81529214

Implementation: 
<img width="1668" alt="Screenshot 2021-09-13 at 14 23 40" src="https://user-images.githubusercontent.com/4890675/133092431-ff5b2e1f-7d4a-4bac-9e67-3379a9a53ee9.png">

<img width="1674" alt="Screenshot 2021-09-13 at 14 23 54" src="https://user-images.githubusercontent.com/4890675/133092594-8b3a79e7-289f-4a8a-8d17-7c396155e993.png">

2. Add "Delete profile?" dialog as per https://marvelapp.com/prototype/h11728e/screen/81529212

Implementation: 
<img width="1672" alt="Screenshot 2021-09-13 at 14 31 51" src="https://user-images.githubusercontent.com/4890675/133092712-36bd3aab-7b7c-47a4-aae5-1630c24ef68d.png">

## Verification Steps
Navigate to /realm-settings page, select the "Client policies" tab, and verify:
 1. Table display a list of client policies.
 2. Selecting <more> icon allows selecting "Delete" button, which display "Delete profile?" dialogue.
 3. Switching between "From view" and "JSON editor" is possible.
 4. Switching to "JSON editor" display client policies JSON in edit mode, "Save" and "Reload" buttons are placed underneath the editor.


## Checklist:
